src/front/js/component/DogList.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react';
+import DogProfile from './DogProfile';
+
+const DogList = ({ userId }) => {
+  const [dogs, setDogs] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchDogProfiles = async () => {
+      try {
+        const response = await fetch(`/users/${userId}/profile`, {
+          headers: {
+            'Authorization': `Bearer ${localStorage.getItem('token')}`
+          }
+        });
+        const data = await response.json();
+        setDogs(data.profiles);
+        setLoading(false);
+      } catch (error) {
+        console.error('Error fetching dog profiles:', error);
+      }
+    };
```

Comment:
use fetchUserProfile from flux file rather than fetchDogProfiles

Reply:
DogList.js now simply calls the actions from Flux